crm_location_nuts/models/crm_lead.py

```diff
@@ -118,7 +118,7 @@ def _onchange_country_id(self):
         }
 
     @api.onchange('state_id')
-    def onchange_state_id_base_location_nuts(self):
+    def onchange_state_id_crm_location_nuts(self):
```

Comment:
`@onchange()` methods must be private:

```suggestion
    def _onchange_state_id_crm_location_nuts(self):
```

Reply:
Done, thx!